packages/core/src/components/fl-spinner/fl-spinner.scss

```diff
@@ -0,0 +1,40 @@
+:host {
+  display: block;
+}
+
+:host(.fl-spinner) {
+  height: 100vh;
+  position: relative;
+
+  &::before {
+    animation: 2s linear infinite fl-spinner;
```

Comment:
Can we configure the speed of the spinner? Maybe add a CSS variable called `--fl-spinner-speed`? Not sure if stencil allows you to configure it through attributes

Reply:
I just tried a couple of different ways and it seems I'm having a similar issue to that of the slider. I can't get my CSS to accept certain inputs when they come as a variable. Also tried setting them directly from javascript but I can't access the ::before context which is an issue.